### PR TITLE
os/booting-on-azure: Style guide fixes: Unwrap. Ref links.

### PR DESCRIPTION
**Pull request by joshix**

### CODE REVIEW

File: os/booting-on-azure.md [hunk 1]
```diff
@@ -1,14 +1,10 @@
 # Running CoreOS on Microsoft Azure
 
-## Choosing a Channel
+## Choosing a channel
```
Comment (jonboulle) [inline os/booting-on-azure.md]: ??

Comment (jonboulle) [inline os/booting-on-azure.md]: whoah, I missed that we moved to sentence case. yay!

Comment (joshix) [inline os/booting-on-azure.md]: Sentence case fits the way these documents tend to use a large number of structural headings.It looks less quaint, and we'll save review time by not trying to figure out the many variations of Title Case (which we retain for titles).

(Watchers, see the [style guide, *Heading style*](https://github.com/coreos/docs/blob/master/STYLE.md#heading-style).)

**Comment by joshix** [comment]
atop and to be synced with #709

**Pull request merged**